docs/source/how_to_guides/inference/how_to_do_bootstrap_inference.ipynb

```diff
@@ -0,0 +1,1038 @@
+       "      <td>85</td>\n",
+       "      <td>15</td>\n",
+       "      <td>rest</td>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>2</td>\n",
+       "      <td>1</td>\n",
+       "      <td>low fat</td>\n",
+       "      <td>88</td>\n",
+       "      <td>30</td>\n",
+       "      <td>rest</td>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>3</td>\n",
+       "      <td>2</td>\n",
+       "      <td>low fat</td>\n",
+       "      <td>90</td>\n",
+       "      <td>1</td>\n",
+       "      <td>rest</td>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>4</td>\n",
+       "      <td>2</td>\n",
+       "      <td>low fat</td>\n",
+       "      <td>92</td>\n",
+       "      <td>15</td>\n",
+       "      <td>rest</td>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   Unnamed: 0  id     diet  pulse  time  kind  constant\n",
+       "0           0   1  low fat     85     1  rest         1\n",
+       "1           1   1  low fat     85    15  rest         1\n",
+       "2           2   1  low fat     88    30  rest         1\n",
+       "3           3   2  low fat     90     1  rest         1\n",
+       "4           4   2  low fat     92    15  rest         1"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df = sns.load_dataset(\"exercise\")\n",
+    "replacements = {\"1 min\": 1, \"15 min\": 15, \"30 min\": 30}\n",
+    "df = df.replace({\"time\": replacements})\n",
+    "df[\"constant\"] = 1\n",
+    "\n",
+    "df.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Doing a very simple bootstrap"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The first thing we need is a function that calculates the bootstrap outcome, given an empirical or re-sampled dataset. The bootstrap outcome is the quantity for which you want to calculate standard errors and confidence intervals. In most applications those are just parameter estimates.\n",
+    "\n",
+    "In our case, we want to regress \"pulse\" on \"time\" and a constant, and the function looks as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def ols_fit(data):\n",
+    "\n",
+    "    y = data[\"pulse\"]\n",
+    "    x = data[[\"constant\", \"time\"]]\n",
+    "    params = sm.OLS(y, x).fit().params\n",
+    "    return params"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The outcome function must return a pandas Series.\n",
```

Comment:
The docstrings suggest that the outcome function can also return an ``array-like`` object not only ``pandas.Series``.

Reply:
Adjusted docstring. Arrays are probably still supported but we do not want to promise that for the future because it's a bad habit.